tests/modules/slimfastq/test.yml

```diff
@@ -0,0 +1,41 @@
+- name: slimfastq test_slimfastq_single_end
+  command: nextflow run tests/modules/slimfastq -entry test_slimfastq_single_end -c tests/config/nextflow.config
+  tags:
+    - slimfastq
+  files:
+    - path: output/slimfastq/test.sfq
+      md5sum: 6a942eeca6c99ee9a9a0cedab5d246f1
+    - path: output/slimfastq/versions.yml
+      md5sum: f52351f5c9e6259af02745c8eae5c780
```

Comment:
Probably best to omit the `versions` md5sum?

Reply:
Sure, can do that.